tests/Unit/Tpay/Resolver/TrustedCertificateResolverTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\CommerceWeavers\SyliusTpayPlugin\Unit\Tpay\Resolver;
+
+use CommerceWeavers\SyliusTpayPlugin\Tpay\Security\Notification\Resolver\TrustedCertificateResolver;
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+
+final class TrustedCertificateResolverTest extends TestCase
```

Comment:
Do we even test this class? 
If yes - is this correct and how to test RuntimeException of this class
If no - should I add this into phpunit.xml exclusions